yarn.lock

```diff
@@ -692,13 +692,6 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
-"@emnapi/runtime@^1.2.0":
```

Comment:
Please freeze lockfile

Reply:
Do you have any recommandation on how to do that ? Is there something to add to yarmrc.yml ?

Reply:
Could be done here https://github.com/incubateur-ademe/nosgestesclimat-site-nextjs/blob/preprod/.github/workflows/run-unit-tests.yaml#L26 with `--frozen-lockfile`